packages/jest-gas-reporter/src/index.ts

```diff
@@ -253,6 +272,35 @@ export class GasReporter implements jest.Reporter {
       }
     }
   }
+
+  async saveResultsToFile(hash: string): Promise<void> {
+    const results = {
+      date: Date.now(),
+      networkName: this.provider.network.name,
+      revision: hash,
+      gasConsumed
+    };
+    const resultsString = JSON.stringify(results, null, 4) + '\n';
+    await fs.appendFile('./gas.json', resultsString, err => {
+      if (err) throw err;
+      console.log('Wrote json to gas.json');
+    });
+    await fs.appendFile(
+      './gas.txt',
+      '\n\n\ndate: ' +
+        Date.now() +
```

Comment:
You could format this with `Date(Date.now()).toString()`
```suggestion
        Date(Date.now()).toString() +
```